src/network/networkd-mptcp.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2020 VMware, Inc. */
+#pragma once
+
+#include <linux/if_link.h>
+
+#include "conf-parser.h"
+#include "networkd-link.h"
+#include "networkd-network.h"
+#include "networkd-util.h"
+
+typedef struct MPTCP {
+        NetworkConfigSection *section;
+        Network *network;
+
+        uint32_t subflows;
+        uint32_t add_addr_accepted;
```

Comment:
If I understand these limits correctly, it seems the limits are global settings, not per-address setting, even it is not per-link setting. Maybe, it is better to move them in `Manager`?